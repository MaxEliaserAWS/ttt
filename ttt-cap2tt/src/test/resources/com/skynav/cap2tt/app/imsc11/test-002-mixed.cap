Lambda字幕V4	DF1+1	SCENE"和文標準"

1	00171912/00172321	ある企業が開発した
				謎のディスク＠幅広［―］＠
2	00172324/00172429	“コードＳ”
3	00172502/00172720	このディスクをめぐり
4	00172727/00173123	複数の国による
				スパイ戦が行われていた
5	00173126/00173702	全容は明らかではなかったが
				各国が火花を散らし＠幅広［―］＠
6	00173715/00174004	攻防は数カ月に及んだ
7	00174017/00174707	最終的に ２カ国だけが
				ディスクの在りかを突き止めた
8	00174712/00175126	スパイのひとり
				ジュリアも情報を入手し＠幅広［―］＠
9	00175212/00175519	隠し場所への潜入に成功する
10	00190515/00190612	もしもし
11	00190803/00190920	やあ 今どこに？	＠斜３
12	00190925/00191115	伯母の家にいるの
13	00191129/00191413	休日なのに忙しいね
14	00191501/00191821	実は 近くのデパートに	＠縦左 ＠中頭
				来ているんだ
15	00191917/00192021	寄っても？	＠縦左 ＠行頭
16	00192203/00192409	ほかの用事があって
17	00192412/00192704	もう出るところなの	＠斜３
18	00192809/00193100	カフェで待っていて	＠縦右 ＠中頭
19	00193309/00193621	伯母さんと
				話してもいいかな
20	00193716/00194013	今日は 具合が悪いみたいなの
21	00194115/00194327	別の機会に紹介するわ
22	00194404/00194701	じゃあ そうしようか
23	00194710/00195119	カフェの場所だけど	＠縦左 ＠行頭
				ハノーバー通りから＠幅広［―］＠
24	00195122/00195629	脇道に入って	＠縦左 ＠行頭
				左折した辺りだよね？
25	00195902/00200117	“Rick's Cafe”よ
26	00200225/00200423	先に行ってるよ
27	00200519/00200720	そうだ　そういえば＠幅広［―］＠
28	00200726/00201023	僕のカードに細工しただろう？
29	00201325/00201424	ジュリア	＠斜３
30	00201526/00201624	どうした？	＠斜３
31	00201717/00201909	いったい何のこと？
32	00202107/00202327	昨日のカードゲームの話だよ
33	00202417/00202607	ズルしただろう？	＠斜３
34	00202706/00202900	見破られたのね
35	00204202/00204305	待って
36	00204308/00204703	やっぱり
				すぐに会いに行くわ
37	00214107/00214427	キーとディスク
				両方が必要だったんだ
