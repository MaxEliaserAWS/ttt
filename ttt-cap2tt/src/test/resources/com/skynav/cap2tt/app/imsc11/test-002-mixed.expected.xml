<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttp:frameRate="30" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttva:model="imsc11" xml:lang="ja" xmlns:ebutts="urn:ebu:tt:style" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" ebutts:multiRowAlign="start" tts:textAlign="center"/><style xml:id="s2" tts:fontSize="1.0em"/><style xml:id="s3" ebutts:multiRowAlign="start" tts:shear="16.78842%" tts:textAlign="center"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/><region xml:id="縦右" tts:extent="15rw 80rh" tts:position="right 10rw center" tts:writingMode="tbrl"/><region xml:id="縦左" tts:displayAlign="after" tts:extent="15rw 80rh" tts:position="left 10rw center" tts:writingMode="tbrl"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:17:19:12" end="00:17:23:21" style="s1">ある企業が開発した<br/>謎のディスク<span style="s2">⸺</span></p><p begin="00:17:23:24" end="00:17:24:29" style="s1">“コードＳ”</p><p begin="00:17:25:02" end="00:17:27:20" style="s1">このディスクをめぐり</p><p begin="00:17:27:27" end="00:17:31:23" style="s1">複数の国による<br/>スパイ戦が行われていた</p><p begin="00:17:31:26" end="00:17:37:02" style="s1">全容は明らかではなかったが<br/>各国が火花を散らし<span style="s2">⸺</span></p><p begin="00:17:37:15" end="00:17:40:04" style="s1">攻防は数カ月に及んだ</p><p begin="00:17:40:17" end="00:17:47:07" style="s1">最終的に ２カ国だけが<br/>ディスクの在りかを突き止めた</p><p begin="00:17:47:12" end="00:17:51:26" style="s1">スパイのひとり<br/>ジュリアも情報を入手し<span style="s2">⸺</span></p><p begin="00:17:52:12" end="00:17:55:19" style="s1">隠し場所への潜入に成功する</p><p begin="00:19:05:15" end="00:19:06:12" style="s1">もしもし</p><p begin="00:19:08:03" end="00:19:09:20" style="s3">やあ 今どこに？</p><p begin="00:19:09:25" end="00:19:11:15" style="s1">伯母の家にいるの</p><p begin="00:19:11:29" end="00:19:14:13" style="s1">休日なのに忙しいね</p><p begin="00:19:15:01" end="00:19:18:21" region="縦左" style="s1">実は 近くのデパートに<br/>来ているんだ</p><p begin="00:19:19:17" end="00:19:20:21" region="縦左">寄っても？</p><p begin="00:19:22:03" end="00:19:24:09" style="s1">ほかの用事があって</p><p begin="00:19:24:12" end="00:19:27:04" style="s3">もう出るところなの</p><p begin="00:19:28:09" end="00:19:31:00" region="縦右" style="s1">カフェで待っていて</p><p begin="00:19:33:09" end="00:19:36:21" style="s1">伯母さんと<br/>話してもいいかな</p><p begin="00:19:37:16" end="00:19:40:13" style="s1">今日は 具合が悪いみたいなの</p><p begin="00:19:41:15" end="00:19:43:27" style="s1">別の機会に紹介するわ</p><p begin="00:19:44:04" end="00:19:47:01" style="s1">じゃあ そうしようか</p><p begin="00:19:47:10" end="00:19:51:19" region="縦左">カフェの場所だけど<br/>ハノーバー通りから<span style="s2">⸺</span></p><p begin="00:19:51:22" end="00:19:56:29" region="縦左">脇道に入って<br/>左折した辺りだよね？</p><p begin="00:19:59:02" end="00:20:01:17" style="s1">“Rick's Cafe”よ</p><p begin="00:20:02:25" end="00:20:04:23" style="s1">先に行ってるよ</p><p begin="00:20:05:19" end="00:20:07:20" style="s1">そうだ　そういえば<span style="s2">⸺</span></p><p begin="00:20:07:26" end="00:20:10:23" style="s1">僕のカードに細工しただろう？</p><p begin="00:20:13:25" end="00:20:14:24" style="s3">ジュリア</p><p begin="00:20:15:26" end="00:20:16:24" style="s3">どうした？</p><p begin="00:20:17:17" end="00:20:19:09" style="s1">いったい何のこと？</p><p begin="00:20:21:07" end="00:20:23:27" style="s1">昨日のカードゲームの話だよ</p><p begin="00:20:24:17" end="00:20:26:07" style="s3">ズルしただろう？</p><p begin="00:20:27:06" end="00:20:29:00" style="s1">見破られたのね</p><p begin="00:20:42:02" end="00:20:43:05" style="s1">待って</p><p begin="00:20:43:08" end="00:20:47:03" style="s1">やっぱり<br/>すぐに会いに行くわ</p><p begin="00:21:41:07" end="00:21:44:27" style="s1">キーとディスク<br/>両方が必要だったんだ</p></div></body></tt>